torch/multiprocessing/spawn.py

```diff
@@ -169,7 +169,7 @@ def join(self, timeout=None):
 class SpawnContext(ProcessContext):
     def __init__(self, processes, error_queues):
         warnings.warn('SpawnContext is renamed to ProcessContext since 1.4 release.')
-        super(SpawnContext, self).__init__(self, processes, error_queues)
+        super(SpawnContext, self).__init__(processes, error_queues)
```

Comment:
Good fix. Looks like no one was using this and `SpawnContext` should be removed - not high-prio though.